server/services/SessionService.js

```diff
@@ -5,14 +5,29 @@ class SessionService {
   constructor() {
     this._session = new Session();
     this._sessionModel = new SessionModel(this._session);
+    this._wrKeys = ['grower_account_id', 'wallet', 'user_photo_url'];
+  }
+
+  formatKeys(filter) {
+    return Object.entries(filter).reduce((obj, [key, val]) => {
+      const copy = { ...obj };
+      if (this._wrKeys.includes(key)) {
```

Comment:
I think we can set the key to wallet_registrations rather than _wrKeys and then having to reformat it

Reply:
I'm not sure I understand your advice, can you please clarify?

Those particular keys need to be designated as related to the wallet_registrations table for the query to work.  We could do a simple conditional to rename them, but formatKeys could become a utility fn() if we need to do similar transformations.

Reply:
I meant just renaming '_wrkeys' to '_walletRegistrationKeys'